File: js/sumPairs.js
const sumPairs = (arr, num) => {
    let ansArr = []
    for (i = 0; i <= arr.length; i++){
        for (j = i; j <= arr.length; j++){
            if (i === j) {
                continue
            } else if (i + j === num){
                ansArr.push(i)
                ansArr.push(j)
            }
        }
    }
    if (ansArr.length === 0){
        return "unable to find pairs"
    } else {
    return ansArr
    }
};

module.exports = sumPairs;

File: python/sum_pairs.py
def sum_pairs(arr, num):
    ans_lst = []
    for i in range(len(arr)):
        for j in range(i + 1, len(arr)):
            if arr[i] + arr[j] == num:
                ans_lst.append([arr[i], arr[j]])
                
    if ans_lst == []:
        return "unable to find pairs"
    else:
        return ans_lst
    
print(sum_pairs([3, 1, 5, 8, 2], 27))

